### DISCUSSION TITLE
Log4j vulnerability

### DISCUSSION THREAD

**Discussion by kevinferns8** [antlr/antlr4#3430] [Q&A]
Hello guys, Any Idea if ANTLR 4 affected by any of the recent log4j security vulnerabilities. Seen an issue saying antl4 uses log4j(https://github.com/antlr/antlr4/issues/3428)

**Comment by parrt** [comment]
I don't remember us using it anywhere but maybe one of the libraries we use for building has it.  Certainly is not in the runtime. :)
